tools/train.py

```diff
@@ -257,6 +257,15 @@ def main(args):
         outputs = trainer(image_infos, cam_infos)
         trainer.update_visibility_filter()
 
+        if step > 0 and step % 5000 == 0:
+            # for class_name, model in trainer.models.items():
+            export_gaussians_to_ply(trainer.models["Background"], cfg.log_dir, f"{args.run_name}_{step}_Background.ply")
+            # pcd = trainer.models["Background"].export_gaussians_to_ply(alpha_thresh=0)
+            # file_name = f"{args.run_name}_{step}_Background.ply"
+            # file_path = f"{cfg.log_dir}/{file_name}"
+            # o3d.io.write_point_cloud(file_path, pcd)
+            print(f"{args.run_name}_{step}_Background.ply stored in {cfg.log_dir}")
+        
```

Comment:
Thanks! I have two suggestions to improve the code:

1. Consider making the export frequency configurable for flexibility:
   ```yaml
   logging:
     # other things remain the same
     export_freq: [7000, 30000]
   ```
   This allows adjusting save frequency per configuration, preventing too many `.ply` files being generated.

   Reference: https://github.com/ziyc/drivestudio/blob/388642a6a998833cb388e0f1a65dfc2071cc6a61/configs/omnire.yaml#L274-L278

2. I suggest moving the relevant code block to the saving operation section, as shown in this screenshot:
   ![image](https://github.com/user-attachments/assets/b4bf6e6c-996f-4a77-b1ec-a5c363ac8611)

   This would improve code organization.